### ISSUE TITLE
Remove jarfiles that cause an exception on reading

### ISSUE THREAD

**Issue by NotActuallyTerry** [LoveTropics/ServerPackLocator#8]

**Comment by sekwah41** [comment]
Should help with tracking down the issue.

Here is the corrupted jar for testing.
[lt-assets.zip](https://github.com/LoveTropics/ServerPackLocator/files/13255614/lt-assets.zip)

```
10:28:34.673
game
at java.base/java.util.stream.ReferencePipeline.toList(ReferencePipeline.java:627)
10:28:34.674
game
at MC-BOOTSTRAP/fml_loader@47.1.47/net.minecraftforge.fml.loading.moddiscovery.AbstractJarFileModLocator.scanMods(AbstractJarFileModLocator.java:21)
10:28:34.674
game
at LAYER SERVICE/serverpacklocator/cpw.mods.forge.serverpacklocator.client.ClientSidedPackHandler.scanMods(ClientSidedPackHandler.java:67)
10:28:34.674
game
at LAYER SERVICE/serverpacklocator/cpw.mods.forge.serverpacklocator.PackLocator.scanMods(PackLocator.java:40)
10:28:34.674
game
at MC-BOOTSTRAP/fml_loader@47.1.47/net.minecraftforge.fml.loading.moddiscovery.ModDiscoverer.discoverMods(ModDiscoverer.java:75)
10:28:34.674
game
at MC-BOOTSTRAP/fml_loader@47.1.47/net.minecraftforge.fml.loading.FMLLoader.beginModScan(FMLLoader.java:164)
10:28:34.674
game
at MC-BOOTSTRAP/fml_loader@47.1.47/net.minecraftforge.fml.loading.FMLServiceProvider.beginScanning(FMLServiceProvider.java:86)
10:28:34.674
game
at MC-BOOTSTRAP/cpw.mods.modlauncher@10.0.9/cpw.mods.modlauncher.TransformationServiceDecorator.runScan(TransformationServiceDecorator.java:112)
10:28:34.674
game
at MC-BOOTSTRAP/cpw.mods.modlauncher@10.0.9/cpw.mods.modlauncher.TransformationServicesHandler.lambda$runScanningTransformationServices$8(TransformationServicesHandler.java:100)
10:28:34.674
game
at java.base/java.util.stream.ReferencePipeline$3$1.accept(ReferencePipeline.java:197)
10:28:34.674
game
at java.base/java.util.HashMap$ValueSpliterator.forEachRemaining(HashMap.java:1779)
10:28:34.674
game
at java.base/java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:509)
10:28:34.674
game
at java.base/java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:499)
10:28:34.674
game
at java.base/java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:575)
10:28:34.674
game
at java.base/java.util.stream.AbstractPipeline.evaluateToArrayNode(AbstractPipeline.java:260)
10:28:34.674
game
at java.base/java.util.stream.ReferencePipeline.toArray(ReferencePipeline.java:616)
10:28:34.674
game
at java.base/java.util.stream.ReferencePipeline.toArray(ReferencePipeline.java:622)
10:28:34.674
game
at java.base/java.util.stream.ReferencePipeline.toList(ReferencePipeline.java:627)
10:28:34.674
game
at MC-BOOTSTRAP/cpw.mods.modlauncher@10.0.9/cpw.mods.modlauncher.TransformationServicesHandler.runScanningTransformationServices(TransformationServicesHandler.java:102)
10:28:34.674
game
at MC-BOOTSTRAP/cpw.mods.modlauncher@10.0.9/cpw.mods.modlauncher.TransformationServicesHandler.initializeTransformationServices(TransformationServicesHandler.java:55)
10:28:34.674
game
at MC-BOOTSTRAP/cpw.mods.modlauncher@10.0.9/cpw.mods.modlauncher.Launcher.run(Launcher.java:88)
10:28:34.674
game
at MC-BOOTSTRAP/cpw.mods.modlauncher@10.0.9/cpw.mods.modlauncher.Launcher.main(Launcher.java:78)
10:28:34.674
game
at MC-BOOTSTRAP/cpw.mods.modlauncher@10.0.9/cpw.mods.modlauncher.BootstrapLaunchConsumer.accept(BootstrapLaunchConsumer.java:26)
10:28:34.674
game
at MC-BOOTSTRAP/cpw.mods.modlauncher@10.0.9/cpw.mods.modlauncher.BootstrapLaunchConsumer.accept(BootstrapLaunchConsumer.java:23)
10:28:34.674
game
at cpw.mods.bootstraplauncher@1.1.2/cpw.mods.bootstraplauncher.BootstrapLauncher.main(BootstrapLauncher.java:141)
```

Taking a look the crash itself is actually within forge due to AbstractJarFileModLocator.scanMods(AbstractJarFileModLocator.java:21)

**Comment by Gegy** [comment]
Resolved in dbe1efc130df78eacbd01b5d601a94ca8b93f517
